contents/blog/cookieless-analytics.md

```diff
@@ -0,0 +1,41 @@
+---
+date: 2021-08-13
+title: Cookieless Analytics - How To Track Website and App Users Without Third-Party Cookies
```

Comment:
This title is factually correct but dull - what about 'Google is about to make it _a lot_ harder for you to do product analytics' or something along those lines?